spec/support/factory/schedules/timetable_factory.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Squadster.Support.Factory.Schedules.TimetableFactory do
+  defmacro __using__(_opts) do
+    quote do
+      def timetable_factory do
+        %Squadster.Schedules.Timetable{
+          date: Faker.Date.backward(5)
```

Comment:
I think it should correspond squads class_day field and maybe check other timetables on the squad to not create timetables with duplicate dates. Also it's better to create timetables for future dates, not for the past